assets/pulpcore-api.prep

```diff
@@ -6,6 +6,9 @@ if -n {
 foreground {
   mkdir -p /etc/pulp/certs
 }
+foreground {
+  dd if=/dev/urandom bs=32 count=1 2>/dev/null | openssl base64 > /etc/pulp/certs/database_fields.symmetric.key
```

Comment:
not sure, urandom is a good choice for an encryption key.
Is `openssl base64` any different from `base64`?
The rest looks good to me.

Reply:
Here's another option I discovered?

```
openssl rand -base64 32
```

Reply:
I was researching how to generate it as securely as possible.

Based on my research, that sounds like a more secure manner:
[OpenSSL 1.1.1d behavior](https://manpages.debian.org/buster/openssl/openssl-rand.1ssl.en.html)
[OpenSSL 3.0 Alpha behavior](https://manpages.debian.org/experimental/openssl/openssl-rand.1ssl.en.html)

Might I suggest we ask the Fenret project for advice, since this is security-related, and they are the experts.

Reply:
The only fernet presence I see online is at [this repo for their spec](https://github.com/fernet/spec/issues) but it looks pretty dead. I have a couple contacts on the security team at Red Hat. Let me reach out to them for advice.